stylus/include/AboutUs.styl

```diff
@@ -0,0 +1,52 @@
+.AboutUs
+	>div:nth-child(1)
+		display: flex
+		padding-bottom: 6rem
+		padding-top: 4rem
+		>div
+			width: 50%
+			>img
+				padding-left: 3rem
+				padding-right: 3rem
+			>div
+				margin-right: 5rem
+			>div:nth-child(1)
+				font-family: Arvo
+				font-size: 2rem
+			>div:nth-child(8)
+				font-weight: bold
+				text-transform: uppercase
+		>div:nth-child(2)
+			padding-top: 3rem
+	>div:nth-child(2)
+		display: flex
+		justify-content: center
+		padding-bottom: 3rem
+		>div:nth-child(2), >div:nth-child(4), >div:nth-child(6)
+			background-color: $Mustard
+			position: relative
+			>img
+				height: 20rem
+				position: relative
+				width: 25rem
+			>div
+				background: $Charcoal
+				color: white
+				font-family: Arvo
+				left: 50%
+				padding-bottom: 1rem
+				padding-left: 1rem
+				padding-right: 1rem
+				padding-top: 1rem
+				position: absolute
+				top: 50%
+				transform: translate(-50%, -50%)
+		>div:nth-child(2):hover, >div:nth-child(4):hover, >div:nth-child(6):hover
+			>img
+				opacity: 0.5
+		>div:nth-child(3), >div:nth-child(5)
+			padding-right: 1rem
+		>div:nth-child(1), >div:nth-child(7)
+			padding-top: 6rem
+			>img
+				height: 7rem
```

Comment:
You're going a little too overboard with the CSS child selectors, feel free to add classes to some of the top level HTML elements.